src/pages/address/components/TokenRow.tsx

```diff
@@ -0,0 +1,281 @@
+import React from "react";
+import LinkText from "@components/commons/LinkText";
+import NumericFormat from "@components/commons/NumericFormat";
+import { Token } from "mockdata/TokenData";
+import { truncateTextFromMiddle } from "shared/textHelper";
+import AddressTokenTableTitle from "./AddressTokenTableTitle";
+
+export default function TokenRow({ data }: { data: Token }) {
+  return (
+    <div>
+      {/* desktop */}
+      <div className="hidden lg:block">
+        <div className="grid grid-cols-9 py-5">
+          <div>
+            <LinkText
+              label={data.asset}
+              href={`/address/${data.asset}`}
+              customStyle="tracking-[0.01em]"
+            />
+          </div>
+          <div>
+            <div className="text-white-50 tracking-[0.01em]">{data.type}</div>
+          </div>
+          <div>
+            <div className="text-white-50 tracking-[0.01em]">
+              {data.symbol.toUpperCase()}
+            </div>
+          </div>
+          <div className="col-span-2 text-right pr-10">
+            <NumericFormat
+              value={data.amount.toString()}
```

Comment:
`value` also accept `number` type now

Reply:
updated removed .toString() for NumericFormat value